### PR TITLE
update history and version number for v0.8.0

### PR DESCRIPTION
**Pull request by wang-boyu**

### CODE REVIEW

**Comment by wang-boyu** [comment]
Will update this PR after RTD is fixed: https://github.com/projectmesa/mesa-geo/pull/223

File: HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,31 @@
 Release History
 ---------------
 
+## 0.8.0 (2024-08-15)
+
+### Special Notes
+
+- Remove Tornado visualization server and replace it with Solara that is compatible with Jupiter notebooks (https://github.com/projectmesa/mesa-geo/pull/212). This is in line with Mesa's recent changes to use Solara for visualization.
```
Comment (EwoutH) [inline HISTORY.md:8]: I would add specifically that 0.8 is the last release series compatible with Mesa 2.3.x, and that Mesa-geo 1.0 will be compatible with Mesa 3.0.

Comment (wang-boyu) [reply HISTORY.md:8]: Updated accordingly

**Comment by wang-boyu** [comment]
Should be ready for review now.

File: HISTORY.md (hunk 1 above)
Comment (EwoutH) [reply HISTORY.md:8]: Thanks!

File: docs/conf.py [hunk 2]
```diff
@@ -66,9 +66,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = "0.7.1"
+version = "0.8.0"
 # The full version, including alpha/beta/rc tags.
-release = "0.7.1"
+release = "0.8.0"
```
Comment (EwoutH) [inline docs/conf.py:71]: We never update this in regular Mesa. Maybe we should start doing this. Good catch.

**Review by EwoutH — APPROVED** [review]

**Comment by wang-boyu** [comment]
There seems to be some additional work on RTD by @tpike3. Pausing this update until that is done.

**Comment by wang-boyu** [comment]
Once again we're ready

File: HISTORY.md [hunk 3]
```diff
@@ -1,6 +1,40 @@
 Release History
 ---------------
 
+## 0.8.0 (2024-08-21)
+
+### Special Notes
+
+- Remove Tornado visualization server and replace it with Solara that is compatible with Jupiter notebooks (https://github.com/projectmesa/mesa-geo/pull/212). This is in line with Mesa's recent changes to use Solara for visualization.
```
Comment (EwoutH) [inline HISTORY.md:8]:
```suggestion
- The Tornado visualization server is removed and replaced with SolaraViz, which is also works within Jupyter notebooks [#212](https://github.com/projectmesa/mesa-geo/pull/212). This is in line with Mesa's recent changes to use Solara for visualization.
```

File: HISTORY.md [hunk 4]
```diff
@@ -1,6 +1,40 @@
...
+- 0.8 is the last release compatible with Mesa 2.3. Next release is 1.0 and it will be compatible with Mesa 3.0.
```
Comment (EwoutH) [inline HISTORY.md:9]: Maybe mention the tutorial
```suggestion
- The [Introductory Tutorial](https://mesa-geo.readthedocs.io/en/stable/tutorials/intro_tutorial.html) has been fully rewritten for Mesa-geo 0.8.0
- The 0.8.x series are the releases compatible with Mesa 2.3.x. The next major release will be compatible with Mesa 3.0+.
```

File: HISTORY.md [hunk 5]
```diff
@@ -1,6 +1,40 @@
 Release History
 ---------------
 
+## 0.8.0 (2024-08-21)
+
+### Special Notes
```
Comment (EwoutH) [inline HISTORY.md:6]: For consistency with Mesa
```suggestion
### Highlights
```

**Review by EwoutH — APPROVED** [review]
Three suggestions, otherwise looks great!

**Comment by EwoutH** [comment]
I would say go ahead and merge and release!

**Comment by wang-boyu** [comment]
Thanks for reviewing this @EwoutH !
@rht @tpike3 any other changes you'd like to see?

**Review by tpike3 — APPROVED** [review]
Thanks @wang-boyu! Looks Great!

**Pull request merged**